dbt/adapters/snowflake/relation_configs/dynamic_table.py

```diff
@@ -81,21 +69,22 @@ def parse_relation_results(cls, relation_results: RelationResults) -> dict:
             "schema_name": dynamic_table.get("schema_name"),
             "database_name": dynamic_table.get("database_name"),
             "query": dynamic_table.get("text"),
+            "target_lag": dynamic_table.get("target_lag"),
             "snowflake_warehouse": dynamic_table.get("warehouse"),
         }
 
-        if dynamic_table.get("target_lag"):
-            config_dict.update(
-                {
-                    "target_lag": SnowflakeDynamicTableTargetLagConfig.parse_relation_results(
-                        dynamic_table
-                    )
-                }
-            )
-
         return config_dict
 
 
+@dataclass(frozen=True, eq=True, unsafe_hash=True)
+class SnowflakeDynamicTableTargetLagConfigChange(RelationConfigChange):
```

Comment:
Nope, this will pass straight through. If we did want validation, we would put it on `SnowflakeDynamicTableConfig`. But for now, we're going without until we settle on exactly what values we're accepting.

Reply:
okay so we are technically allowing any string value to be passed for now alright will finish up tests and then this should be ready for merger

Reply:
> okay so we are technically allowing any string value
Correct, we're relying on the Snowflake check for now. We'll come back around and put some validation in later, probably in 1.7.